charts/gitops-runtime/Chart.yaml

```diff
@@ -15,11 +15,15 @@ annotations:
   artifacthub.io/alternativeName: "codefresh-gitops-runtime"
   artifacthub.io/changes: |
     - kind: changed
-      description: Update `argo-events` to 2.0.5-1-cf-init
```

Comment:
We accumulate changes in the changelog - So the upgrade of Argo-Events is also a part of this release. Just add another member to the changes array